app/src/main/java/ncnf/main/MainActivity.java

```diff
@@ -1,20 +1,39 @@
-package com.example.bootcamp;
+package ncnf.main;
 
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.View;
+import android.widget.Button;
 import android.widget.EditText;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.ncnf.R;
+
+import ncnf.feed.FeedActivity;
 
 public class MainActivity extends AppCompatActivity {
 
     public static final String EXTRA_MESSAGE = "com.example.bootcamp.EXTRA_MESSAGE";
 
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        RecyclerView r = (RecyclerView)findViewById(R.id.recycler_view);
+
+        Button launchFeed = (Button) findViewById(R.id.feed_view_button);
+
+        launchFeed.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+
+                feedView();
+            }
+        });
```

Comment:
Is it not cleaner/simpler to to add the click event directly in activity view ? (android:onClick="goToFeed")